### ISSUE TITLE
it seems that languageLevel didn't work as expected ??

### ISSUE THREAD

**Issue by haoliangwu** [dherges/ng-packagr#566]
## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description
this tool was really cool, but I don't know if I used it in right way, it seems that ``languageLevel`` config didn't work as expected, something like ``"languageLevel": [ "dom", "es2017" ]``, but ng-packagr always gave me error below: 

```
BUILD ERROR
Error at /Users/lyon/lyon/ngx-anchor/.ng_pkg_build/ngx-anchor/ts/src/app/ngx-anchor/utils/dom.ts:92:28: Property 'includes' does not exist on type 'any[]'.
Error: Error at /Users/lyon/lyon/ngx-anchor/.ng_pkg_build/ngx-anchor/ts/src/app/ngx-anchor/utils/dom.ts:92:28: Property 'includes' does not exist on type 'any[]'.
```

this was because I use ``Array.includes`` which provided in es2016, however with the ``"languageLevel": [ "dom", "es2017" ]`` it should works.

Originally, I used ``ngc`` + ``rollup`` to bundle it and it works fine.

#### How To Reproduce
I created an issue demo branch from my library repo, https://github.com/haoliangwu/ngx-anchor/tree/issue

#### Expected Behaviour

it should build successfully.

#### Version Information

```
ng-packagr: 1.6.0
@angular/*: 5.2.1
typescript: 2.6.2
node: 8.9.3
npm: 5.6.0
```

**Comment by dherges** [comment]
Yes, it's implemented as tsconfig pass through. My guess it's that you need to pass `["dom", "es2015", "es2016", "es2017"] then?!?

https://github.com/dherges/ng-packagr/blob/master/src/lib/ts/init-tsconfig.ts#L32

My personal preference would be:

> I, as a library author, like to write source code against ESx features, so that I can use the new and shiny APIs. From blog posts and things that I've read, people taught me that es2016 is a superset of es2015 and es2017 is a superset of es2016. Thus, I'd like to specify the language level with an ordered enum, i.e., one of "es2017", "es2016", "es2015". I expect the packaging tool to correctly set up the transpiler's config.

**Comment by dherges** [comment]
@sumigoma wdyt? #486

**Comment by sumigoma** [comment]
`"languageLevel": [ "dom", "es2017" ]` should work. @haoliangwu this change landed in `ng-packagr@2.0.0-rc.11`, but it appears you are trying to use `1.6.0`.

**Comment by haoliangwu** [comment]
@sumigoma @dherges thx for answering, I'll add ``Array.includes`` polyfill manually as workaround until the  ``ng-packagr@2.0.0`` finally released. 

thx for your hard work.
